src/main/resources/org/opensearch/system/applicationtemplates/v1/k8s-logs.json

```diff
@@ -2830,7 +2830,8 @@
   },
   "_meta": {
     "_type": "@abc_template",
-    "_version": 1
+    "_version": 1,
+    "_schema_ref": "https://github.com/opensearch-project/opensearch-catalog/blob/main/integrations/observability/k8s/schemas"
```

Comment:
I've changed to point to the file link which contains components from which mappings are built for specific templates. Since there are multiple mappings which are combined in the catalog repo to build the final opensearch template, there is not a single file to refer to. Hence, I've referred to the version through the manifest file for each integration in the catalog repo.